breathecode/media/models.py

```diff
@@ -40,6 +40,8 @@ class MediaResolution(models.Model):
     width = models.IntegerField()
     height = models.IntegerField()
     hits = models.IntegerField(default=0)
+    
+    media = models.ForeignKey(Media, on_delete=models.CASCADE, blank=True, null=True)
```

Comment:
We need also in `MaskingUrlView` we need that the first academy that resize a image, and `media=None`, the first academy claim this `MediaResolution`

Reply:
In production, all the media resolution it's `media=None` because a need field will be add to the media resolution when the heroku perform the migration, this media resolution should stay orphans without it